tests/extmod/urandom_extra.py

```diff
@@ -69,11 +69,21 @@
     print('IndexError')
 
 print('random')
-for i in range(50):
-    assert 0 <= random.random() < 1
+try:
+    eval('1.0')  # is float supported?
+except SyntaxError:
+    pass
+else:
+    for i in range(50):
+        assert 0 <= random.random() < 1
```

Comment:
You'll see that `ujson` float tests are split out into separate files.  Let's do the same here: move all float related functions from this file into `urandom_extra_float.py`

Reply:
Done. That's a lot nicer.